### ISSUE TITLE
does lstm only running on CPU? Can it delegated to Apple Neural Engine(ANE) by CoreML delegate ?

### ISSUE THREAD

**Issue by xunbing** [tensorflow/tensorflow#51674]
Such as title. 
And I have another confuse. When I split my model into two pieces, the second piece consists of two nodes(Dense and Softmax) can not be delegate to CoreML anymore.

who can tell me something about this? Any reply will be appreciate.

**Comment by xunbing** [comment]
@tilakrayal  thanks a lot! In these days, I'm trying my best to figure it out. The two blogs are very helpful to me.
